src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/RadioButtonsContainerComponent.tsx

```diff
@@ -83,19 +83,24 @@ export const RadioButtonContainerComponent = (props: IRadioButtonsContainerProps
   const options = apiOptions || props.options || [];
   const radioGroupIsRow: boolean = (options.length <= 2);
 
+  React.useEffect(() => {
+    returnSelected();
+  }, [useSelector((state: IRuntimeState) => state.optionState.options[props.optionsId])]);
```

Comment:
Could this be rewritten to 
```
 React.useEffect(() => {
    returnSelected();
  }, [options]); 
```

Reply:
Also: bump app-frontend version 🗡️

Reply:
that was my inital attempt, but I couldn't get it working.

Reply:
But it works now, so simplifying it ;)